spec/models/medical_safety_alert_spec.rb

```diff
@@ -0,0 +1,175 @@
+      },
+      "routes" => [
+        {
+          "path" => "/drug-device-alerts/example-medical-safety-alert-#{n}",
+          "type" => "exact",
+        }
+      ],
+      "redirects" => [],
+      "update_type" => "major",
+    }
+  end
+
+  let(:non_medical_safety_alert_content_item) {
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/other-documents/not-a-medical-safety-alert",
+      "format" => "specialist_document",
+      "details" => {
+        "metadata" => {
+          "document_type" => "not_an_medical_safety_alert",
+        },
+      },
+    }
+  }
+
+  let(:mhra_org_content_item) {
+    {
+      "base_path" => "/government/organisations/medicines-and-healthcare-products-regulatory-agency",
+      "content_id" => "240f72bd-9a4d-4f39-94d9-77235cadde8e",
+      "title" => "Medicines and Healthcare products Regulatory Agency",
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-12-01T15:51:43.521Z",
+      "public_updated_at" => "2015-06-29T14:33:29.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "EA63",
+      "links" => {
+        "available_translations" => [
+          {
+            "content_id" => "240f72bd-9a4d-4f39-94d9-77235cadde8e",
+            "title" => "Medicines and Healthcare products Regulatory Agency",
+            "base_path" => "/government/organisations/medicines-and-healthcare-products-regulatory-agency",
+            "description" => nil,
+            "api_url" => "https://www.gov.uk/api/content/government/organisations/medicines-and-healthcare-products-regulatory-agency",
+            "web_url" => "https://www.gov.uk/government/organisations/medicines-and-healthcare-products-regulatory-agency",
+            "locale" => "en"
+          }
+        ]
+      },
+      "description" => nil,
+      "details" => {}
+    }
+  }
+
+  let(:indexable_attributes) {
+    {
+      "title" => "Example Medical Safety Alert 0",
+      "description" => "This is the summary of example Medical Safety Alert 0",
+      "link" => "/drug-device-alerts/example-medical-safety-alert-0",
+      "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example Medical Safety Alert" * 10),
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "organisations" => ["medicines-and-healthcare-products-regulatory-agency"],
+      "alert_type" => "company-led-drugs",
+      "issued_date" => "2016-02-01",
+    }
+  }
+
+  let(:fields) { %i[base_path content_id] }
+
+  let(:medical_safety_alerts) { 10.times.map { |n| medical_safety_alert_content_item(n) } }
+
+  before do
+    publishing_api_has_fields_for_format('specialist_document', medical_safety_alerts, fields)
+
+    medical_safety_alerts.each do |medical_safety_alert|
+      publishing_api_has_item(medical_safety_alert)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  describe "#all" do
```

Comment:
Ugh. I'll make a note to update where I copied and pasted them from then as well as here and presumably https://github.com/alphagov/specialist-publisher/pull/656/files#diff-87a2b2747a4f709e7ef875915e7fcee5R106 too.